prow/config.yaml

```diff
@@ -306,17 +306,20 @@ tide:
     - istio/release-builder
     labels:
     - "cla: yes"
-    missingLabels:
+    missingLabels: &istio_tide_missing_labels
     - do-not-merge
     - do-not-merge/hold
     - do-not-merge/work-in-progress
     reviewApprovedRequired: true
+  - orgs:
+      - istio-private
+    missingLabels: *istio_tide_missing_labels
```

Comment:
This config will merge any PR in the org as soon as the tests are passing if there isn't a `do-not-merge/*` label.  In other words Tide won't wait for code review.

If you include `reviewApprovedRequired: true`, Tide will wait for an approve review before merging. However, Tide uses the search token `review: approved` to search for approved PRs. I don't think that it matches PRs that have only been approved by non-CODEOWNERS though so until we deal with the non-applicable CODEOWNERS file we probably can't use that.

If we don't want to worry about the CODEOWNERS problem it might be easiest to use the `lgtm` plugin configured to respect github approve reviews and have Tide require that label?

Reply:
> This config will merge any PR in the org as soon as the tests are passing if there isn't a `do-not-merge/*` label. In other words Tide won't wait for code review. If you include `reviewApprovedRequired: true`, ...

Good catch, I updated to include `reviewApprovedRequired: true`.

Reply:
I think you can actually turn off "require codeowners approval". Then anyone can approve even if they aren't a codeowners, which is probably relevant here

Reply:
> I think you can actually turn off "require codeowners approval". Then anyone can approve even if they aren't a codeowners, which is probably relevant here

Good idea. We should try this first. Let me change that in the `branch-protection` settings and update the PR.